lib/IDE/SyntaxModel.cpp

```diff
@@ -1556,3 +1565,12 @@ bool ModelASTWalker::findFieldsInDocCommentBlock(SyntaxNode Node) {
   std::match_results<StringRef::iterator> Matches;
   return true;
 }
+
+CharSourceRange ModelASTWalker::findDocCommentRange(DeclAttributes Attrs) {
+  for (auto Attr : Attrs) {
+    if (auto DocAttr = dyn_cast_or_null<RawDocCommentAttr>(Attr)) {
+      return DocAttr->getCommentRange();
+    }
+  }
+  return CharSourceRange();
+}
```

Comment:
We really don't need a member function here. A static function in the file will be sufficient.